0000-template.md

```diff
@@ -0,0 +1,39 @@
+# RFC-0000: Feature Name Here
```

Comment:
Usually in RFCs processes you leave the number at 0 and a number gets assigned right before merging.
Since fellowship members are supposed to vote on the hash of the text, I think it should be mentioned how that works here. Should they vote on the text where the number is 0?